### PR TITLE
Fix swagger-ui branding when using gradle

### LINKED ISSUE #39523: quarkus-swagger-ui logo.png not loaded
### Describe the bug

logo.png not loaded by io.quarkus:quarkus-swagger-ui into swagger ui docs

### Expected behavior

In the docs it is stated:

> Logo
To supply your own logo, you need to place a file called logo.png in src/main/resources/META-INF/branding.

See https://quarkus.io/guides/openapi-swaggerui#logo
I did so
image
So I expect that image to be reflected in swagger ui.

### Actual behavior

image
image
It is not loaded tho

### How to Reproduce?

Using a static open api file 
image

application.properties content:

quarkus.swagger-ui.always-include=true
quarkus.swagger-ui.path=docs

mp.openapi.scan.disable=true
quarkus.smallrye-openapi.info-title=Open Api for car insurance api

### Output of `uname -a` or `ver`

_No response_

### Output of `java -version`

21.0.2

### Quarkus version or git rev

quarkusPlatformVersion=3.8.2

### Build tool (ie. output of `mvnw --version` or `gradlew --version`)

_No response_

### Additional information

Tested that `quarkus.swagger-ui.path` makes no difference.

### PR DESCRIPTION
**Pull request by phillip-kruger**
Fix #39523

### CODE REVIEW

File: extensions/vertx-http/deployment/src/main/java/io/quarkus/vertx/http/deployment/webjar/WebJarUtil.java [hunk 1]
```diff
@@ -31,7 +31,8 @@ public class WebJarUtil {
 
     private static final Logger LOG = Logger.getLogger(WebJarUtil.class);
 
-    private static final String CUSTOM_MEDIA_FOLDER = "META-INF/branding/";
+    private static final String CUSTOM_MEDIA_FOLDER_MVN = "META-INF/branding/";
+    private static final String CUSTOM_MEDIA_FOLDER_GRADLE = "META-INF/resources/branding/";
```
Comment (gsmet) [inline extensions/vertx-http/deployment/src/main/java/io/quarkus/vertx/http/deployment/webjar/WebJarUtil.java:35]: It it expected that the structure of the jar changes when using another build tool? I would have expected the conventions to be the same?

/cc @aloubyansky @snazy @glefloch

Comment (snazy) [reply extensions/vertx-http/deployment/src/main/java/io/quarkus/vertx/http/deployment/webjar/WebJarUtil.java:35]: Yea - that should definitely not be the case.
Quarkus (even old Quarkus 2 apps built via Gradle) pulls web-resources from `META-INF/resources/` ([our web-resources](https://github.com/projectnessie/nessie-ui/blob/e75149978e5884533d151a2176c2f78fdb4a5c04/nessie-ui/build.gradle.kts#L28)), but branding-stuff works fine [in `META-INF/branding` for us](https://github.com/projectnessie/nessie/tree/main/servers/quarkus-server/src/main/resources/META-INF/branding).
Maybe the Gradle-build for the app is wrong?

Comment (phillip-kruger) [reply extensions/vertx-http/deployment/src/main/java/io/quarkus/vertx/http/deployment/webjar/WebJarUtil.java:35]: I don't know enough about Gradle, can you check the reproducer shared in the issue ?

Comment (snazy) [reply extensions/vertx-http/deployment/src/main/java/io/quarkus/vertx/http/deployment/webjar/WebJarUtil.java:35]: hehe - the issue is in the issue description - the the 1st screenshot shows `resources/META-INF/resources/branding/logo.png` - that should be `.../META-INF/resources/branding/logo.png`

**Comment by phillip-kruger** [comment]
Thanks @snazy !

**Pull request closed without merging**